src/components/Analysis/Taxonomy/v2index.tsx

```diff
@@ -0,0 +1,175 @@
+          <fieldset
+            data-cy="ssu-lsu-btns"
+            className="vf-form__fieldset vf-stack vf-stack--400 mg-fieldbox"
+            style={{
+              width: '15rem',
+            }}
+          >
+            <legend className="vf-form__legend">rRNA</legend>
+            <label>
+              <input
+                type="radio"
+                id="smallrRNA"
+                data-cy="ssu-btn"
+                name="tax-results"
+                disabled={!enableSSU}
+                value="/ssu"
+                checked={taxResults === '/ssu'}
+                onChange={handleTaxChange}
+              />{' '}
+              small subunit rRNA
+            </label>
+            <label>
+              <input
+                type="radio"
+                id="largerRNA"
+                data-cy="lsu-btn"
+                name="tax-results"
+                disabled={!enableLSU}
+                value="/lsu"
+                checked={taxResults === '/lsu'}
+                onChange={handleTaxChange}
+              />{' '}
+              large subunit rRNA
+            </label>
+          </fieldset>
+        )}
+        {(enableITSoneDB || enableITSUnite) && (
+          <fieldset
+            className="vf-form__fieldset vf-stack vf-stack--400 mg-fieldbox"
+            style={{
+              width: '15rem',
+            }}
+          >
+            <legend className="vf-form__legend">ITS</legend>
+            <label>
+              <input
+                type="radio"
+                id="smallrRNA"
+                name="tax-results"
+                value="/itsonedb"
+                checked={taxResults === '/itsonedb'}
+                onChange={handleTaxChange}
+              />{' '}
+              ITS with ITSoneDB
+            </label>
+            <label>
+              <input
+                type="radio"
+                id="largerRNA"
+                name="tax-results"
+                value="/unite"
+                checked={taxResults === '/unite'}
+                onChange={handleTaxChange}
+              />{' '}
+              ITS1/2 with UNITE
+            </label>
+          </fieldset>
+        )}
+      </div>
+      <div>
+        These are the results from the taxonomic analysis steps of our pipeline.
+        You can switch between different views of the data using the menu of
+        icons below (pie, bar, stacked and interactive krona charts). The data
+        used to build these charts can be found under the{' '}
+        <Link to="#download">Download</Link> tab.
+      </div>
+      <TabsForQueryParameter
+        tabs={tabs}
+        queryParameter={PARAMETER_NAME}
+        defaultValue={PARAMETER_DEFAULT}
+      />
+      <div className="vf-tabs-content">
+        {type === 'krona' ? (
+          <object
+            className="krona_chart"
+            data-cy="krona-chart"
+            data={`${config.api}analyses/${accession}/krona${taxResults}?collapse=false`}
```

Comment:
This one will also change – to a URL that will probably be something like `http://ftp.ebi.ac.uk/public/databases/......`, where that URL will be determined from some finding an item in a JSON list at `apiv2/analyses/${accession}/downloads`. I am working on adding that bit too, so this is just an FYI we will need to update this.

Reply:
I mis-remembered... 
The analysis detail endpoint already has this:
```json
{
  "study_accession": "MGYS000000001",
  "accession": "MGYA000000001",
  "experiment_type": "METAT",
  "downloads": [
    {
      "path": "string",
      "alias": "string",
      "download_type": "Sequence data",
      "file_type": "fasta",
      "long_description": "string",
      "short_description": "string",
      "parent_identifier": 0,
      "url": "string"
    }
  ],
  "run_accession": "ERR0000001",
...
```
We just need to look through the `downloads[]` list for the item with `${taxResults} Krona` in its `short_description`, and use that URL.